pipe/psf_phot.py

```diff
@@ -1316,10 +1326,11 @@ def sa2im_crop(self, data):
         """Crop out the imagette region from a subarray. Works for
         both frames and cubes.
         """
-        i0 = self.im_sa_off[0]
-        i1 = i0+self.im_raw[0].shape[1]
-        j0 = self.im_sa_off[1]
-        j1 = j0+self.im_raw[0].shape[0]
+        i0 = int(self.im_sa_off[0]) - 1
+        i1 = int(i0+self.im_raw[0].shape[1])
+        j0 = int(self.im_sa_off[1]) - 1
+        j1 = int(j0+self.im_raw[0].shape[0])
+

```

Comment:
I found that I couldn't get the tests to run with these values as floats, they needed to be cast to integers, and int he case of the `*0` terms, I had to subtract one to get the shapes to match. Does this make sense to you, @alphapsa?

Reply:
I think the indices _should_ be integers, they should be cast already in read.imagette_offset

Reply:
Without the `-1`, I'm getting the following error: 
```
self = <pipe.psf_phot.PsfPhot object at 0x124a52990>
    def define_im_mask(self):
        """Crops the imagette part of the subarray mask to define the
        imagette mask.
        """
>       self.im_mask = self.sa2im_crop(self.sa_mask) * self.im_apt
E       ValueError: operands could not be broadcast together with shapes (199,199) (200,200)
```